cmd/model/test.go

```diff
@@ -77,7 +76,7 @@ var modelTestCmd = &cobra.Command{
 		summaries := []string{}
 
 		for _, file := range fileNames {
-			format, storeData, err := storetest.ReadFromFile(file, path.Dir(file))
+			format, storeData, err := storetest.ReadFromFile(file, "")
```

Comment:
Tempted to scrap the second arg as both callsites pass an empty string now anyways